lib/features/mailbox_dashboard/presentation/controller/mailbox_dashboard_controller.dart

```diff
@@ -163,13 +163,9 @@ class MailboxDashBoardController extends ReloadableController {
 
   @override
   void onReady() {
-    log('MailboxDashBoardController::onReady()');
-    dispatchRoute(DashboardRoutes.thread);
     _registerPendingEmailAddress();
     _registerPendingFileInfo();
-    _setSessionCurrent();
-    _getUserProfile();
-    _getVacationResponse();
```

Comment:
I prefer separate method like this

Reply:
It is decoupled, but it is not called in the current position. We need to check the input parameters before calling them.